packages/fxa-settings/src/pages/SigninBounced/index.tsx

```diff
@@ -0,0 +1,129 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps, Link, useNavigate } from '@reach/router';
+import { logPageViewEvent } from '../../lib/metrics';
+import { ReactComponent as EmailBounced } from './graphic_email_bounced.svg';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useFtlMsgResolver } from '../../models/hooks';
+
+export type SigninBouncedProps = {
+  onBackButtonClick?: (
+    event: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => void;
+  canGoBack?: boolean;
+  email: string;
+};
+
+const SigninBounced = ({
+  canGoBack,
+  email,
+  onBackButtonClick = () => window.history.back(),
+}: SigninBouncedProps & RouteComponentProps) => {
+  logPageViewEvent('signin-bounced', {
+    entrypoint_variation: 'react',
+  });
+  const ftlMessageResolver = useFtlMsgResolver();
+  const backText = ftlMessageResolver.getMsg('back', 'Back');
+  const navigate = useNavigate();
+
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const emailBouncedAltText = ftlMsgResolver.getMsg(
+    'email-bounced-description',
+    'An orange envelope with a lock printed on the flap is shown bouncing across pale gray clouds'
+  );
```

Comment:
I feel this image is decorative, and it might not be necessary to provide alt text? Generally if an image is "eye candy", it doesn't provide important information that would need to be read out by a screen reader. What's descriptive and what isn't is a bit subjective though!

Reply:
Works for me! I've found that we seem to add alt text onto decorative images pretty frequently on this team, but what you're describing is my preference also! I'll remove it!